webapp/api/advantage.py

```diff
@@ -144,4 +144,7 @@ def accept_renewal(session, renewal_id):
         raise_http_errors=False,
     )
 
-    return response.json()
+    if response.status_code == 200:
```

Comment:
```suggestion
    if response.ok:
```
In case they start introducing redirects or else.
Doc: https://2.python-requests.org/en/master/api/#requests.Response.ok